cloud-gcp/src/main/java/com/sequenceiq/cloudbreak/cloud/gcp/GcpPlatformResources.java

```diff
@@ -444,6 +450,9 @@ public CloudVmTypes getCloudVmTypes(CloudCredential cloudCredential, Region regi
                     }
                 }
 
+                types = types.stream()
+                        .filter(e -> minimalHardwareFilter.suitableAsMinimumHardware(e))
+                        .collect(Collectors.toSet());
```

Comment:
This change will also apply to DL clusters (as called from `virtualMachines()`). Is that what you want?